### PR TITLE
New package: Opsfolio.Surveilr version 0.9.0

### PR DESCRIPTION
**Pull request by Lilit0x**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/155614)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @Lilit0x,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 881.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335187
> 2024-05-29 15:20:10.767 [FAIL] Installer failed security check. Url: https://github.com/opsfolio/releases.opsfolio.com/releases/download/0.9.0/surveilr.exe Result: 0x80004005

Automated error analysis suggests -1978335187 may mean Installer failed security check. Another script should retry this every 18 hours. 

(Automated response - build 881.)

**Comment by Lilit0x** [comment]
@microsoft-github-policy-service agree [company="{Opsfolio}"]

**Comment by Lilit0x** [comment]
@microsoft-github-policy-service agree company="Opsfolio"

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335187
> 2024-05-29 12:53:01.666 [FAIL] Installer failed security check. Url: https://github.com/opsfolio/releases.opsfolio.com/releases/download/0.9.0/surveilr.exe Result: 0x80004005

Automated error analysis suggests -1978335187 may mean Installer failed security check. Another script should retry this every 18 hours. `n`n

(Automated response - build 881.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> ADO Build not found.

(Automated response - build 886.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 886.)

File: manifests/o/Opsfolio/Surveilr/0.9.0/Opsfolio.Surveilr.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Opsfolio.Surveilr
+PackageVersion: 0.9.0
```
Comment (stephengillie) [inline manifests/o/Opsfolio/Surveilr/0.9.0/Opsfolio.Surveilr.installer.yaml:5]:
```suggestion
PackageVersion: 0.9.1
```
Please update file path to match this change.

File: manifests/o/Opsfolio/Surveilr/0.9.0/Opsfolio.Surveilr.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Opsfolio.Surveilr
+PackageVersion: 0.9.0
```
Comment (stephengillie) [inline manifests/o/Opsfolio/Surveilr/0.9.0/Opsfolio.Surveilr.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.9.1
```
Please update file path to match this change.

File: manifests/o/Opsfolio/Surveilr/0.9.0/Opsfolio.Surveilr.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: Opsfolio.Surveilr
+PackageVersion: 0.9.0
```
Comment (stephengillie) [inline manifests/o/Opsfolio/Surveilr/0.9.0/Opsfolio.Surveilr.yaml:5]:
```suggestion
PackageVersion: 0.9.1
```
Please update file path to match this change.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @Lilit0x,

image

This appaers to be version `0.9.1`. (The package manager adds the Registry entry for portable applications.)

**Pull request closed without merging**